main.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+import time
+from temporalio.client import Client
+from temporalio.worker import Worker
+from poker_workflow import PokerWorkflow
+from hand_ranking_workflow import HandRankingWorkflow
+
+
+async def main():
+    # Start client
+    client = await Client.connect("localhost:7233")
+
+    async with Worker(
+        client,
+        task_queue="poker-hand-ranking-task-queue",
+        workflows=[PokerWorkflow, HandRankingWorkflow],
+    ):
+        seed = int(time.time())
+        await client.execute_workflow(
```

Comment:
O: May consider, for larger samples like this, separating the worker process and the starter that waits for workflow completion.

Reply:
documentation link, please?

Reply:
Done.